src/loc/lcl/PTB/Axe.Windows.Rules.lcl

```diff
@@ -2180,10 +2180,13 @@
       </Item>
       <Item ItemId=";NameIsInformative" ItemType="0" PsrId="211" InstFlg="true" Leaf="true">
         <Str Cat="Text">
-          <Val><![CDATA[Provide a UI Automation Name property for the element that:]D;]A; · Concisely identifies the element, AND]D;]A; · Does not include the element's class name (such as Microsoft.*.* or Windows.*.*).]]></Val>
+          <Val><![CDATA[Examine the UI Automation Name property of this element:]D;]A; · If the Name property contains "Microsoft" or "Windows" and is closely related to the on-screen text (for example, a reference to "Microsoft.com"), you may safely ignore this issue.]D;]A; · If the Name property is not closely related to the on-screen text (and provides no meaningful information to end-users), update the Name to a concise, user-facing label that is closely related to the on-screen text.]]></Val>
           <Tgt Cat="Text" Stat="Loc" Orig="New">
-            <Val><![CDATA[Forneça uma propriedade Name de Automação da Interface do Usuário para o elemento que:]D;]A; · Identifique o elemento com concisão, E]D;]A; · Não inclua o nome de classe do elemento (como Microsoft.*.* ou Windows.*.*).]]></Val>
+            <Val><![CDATA[Examine a propriedade Nome da Automação da Interface do Usuário deste elemento:]D;]A; · Se a propriedade Nome contiver "Microsoft" ou "Windows" e estiver relacionada ao texto na tela (por exemplo, uma referência a "Microsoft.com"), você poderá ignorar esse problema com segurança.]D;]A; · Se a propriedade Nome não estiver relacionada ao texto na tela (e não fornecer informações significativas para os usuários finais), atualize o Nome para um rótulo conciso voltado ao usuário que seja mais relacionado ao texto na tela.]]></Val>
```

Comment:
As indicated in the translator comments, `Name` is a UI Automation property that should never be translated: ```suggestion
            <Val><![CDATA[Examine a propriedade Name da Automação da Interface do Usuário deste elemento:]D;]A; · Se a propriedade Name contiver "Microsoft" ou "Windows" e estiver relacionada ao texto na tela (por exemplo, uma referência a "Microsoft.com"), você poderá ignorar esse problema com segurança.]D;]A; · Se a propriedade Name não estiver relacionada ao texto na tela (e não fornecer informações significativas para os usuários finais), atualize o Name para um rótulo conciso voltado ao usuário que seja mais relacionado ao texto na tela.]]></Val>
```